instance/templates/instance/ansible/swift.yml

```diff
@@ -1,8 +1,26 @@
-# The following line is commented out until the openstack role has been merged upstream.
-# EDXAPP_DEFAULT_FILE_STORAGE: 'swift.storage.SwiftStorage'
+EDXAPP_SETTINGS: 'openstack'
+XQUEUE_SETTINGS: 'openstack_settings'
+COMMON_VHOST_ROLE_NAME: 'openstack'
```

Comment:
@haikuginger The configuration repo as of [eucalyptus/1rc1](https://github.com/open-craft/configuration/tree/eucalyptus/1rc1) doesn't use this variable (it's not referenced anywhere). It has been replaced by [`VHOST_NAME`](https://github.com/open-craft/configuration/blob/eucalyptus/1rc1/playbooks/roles/vhost/defaults/main.yml#L16), so that's probably what we should be using here.

We might need to keep `COMMON_VHOST_ROLE_NAME` to stay backward compatible for a little longer (until we've migrated all of our instances to Eucalyptus). Please check before removing it.